.github/workflows/powershell-analyzer.yml

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Analyze PowerShell
+
+on:
+  workflow_call:
+
+jobs:
+  analyze-powershell:
+    runs-on: ubuntu-latest
+    permissions:
+      # required for all workflows
+      security-events: write
+      # only required for workflows in private repositories
+      actions: read
+      contents: read
+
+    steps:
+      - name: Checkout the Actions repo
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846  # v3.0.0
+        with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-Actions
+          path: Ed-Fi-Actions
```

Comment:
I'm curious: why checkout the repository instead of just using the repository's action? The latter is the approach in https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Actions/tree/main/docs. I don't really have a strong preference though, since this works perfectly well. And there might even be a compelling reason for the approach of checking out the repo.

Reply:
Maybe there's something I'm not following, but I see that the repository scanner is calling the yml file https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Actions/blob/main/.github/workflows/repository-scanner.yml also does the checkout the same way, is there a difference on those two configurations?

Reply:
You're right - I was reading this as an _example_ of how _another_ repo would call the action. Just wasn't paying close enough attention. All good here.